crates/common/tedge_config/src/tedge_config_defaults.rs

```diff
@@ -7,6 +7,8 @@ use std::path::Path;
 const DEFAULT_ETC_PATH: &str = "/etc";
 const DEFAULT_PORT: u16 = 1883;
 const DEFAULT_TMP_PATH: &str = "/tmp";
```

Comment:
`DEFAULT_TMP_PATH` could also be made `pub` and fixed so that all the cases it is called from this file, but it is beyond this ticket scope. 

@didier-wenzek  I don't mind making the changes, but is it best in another PR?
The same goes for `DEFAULT_ETC_PATH` and `DEFAULT_PORT`

Reply:
What bother me is that we have so many places using hardcoded `/tmp` or `/etc` paths. Having a `pub const` will change nothing on the main issue: one must avoid hardcoded paths.

Change nothing for now.